iothub/service/src/DigitalTwin/Models/GetDigitalTwinResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// The service response to a get digital twin request.
+    /// </summary>
+    /// <typeparam name="T">The type of the digital twin.</typeparam>
+    public class GetDigitalTwinResponse<T>
```

Comment:
Is it just me or does "**Get**DigitalTwinResponse" seem like a weird name for a data object? The "Get" verb in here is throwing me off since on first glance this looks more like an API call than a data object.

Reply:
I can see where you are coming from. Do you have any suggestions on a possible rename? DigitalTwinWithVersion?

Reply:
I'm ok with DigitalTwinWithVersion. We also have the update variant, what would we rename that to? UpdateDigitalTwinResponse -> DigitalTwinUpdatedResponse?

Reply:
DigitalTwinUpdatedResponse contains only the ETag and location of the digital twin, so following the naming pattern would give us something like DigitalTwinWithVersionAndLocation which sounds clunky to me.

I originally had these return types named like "DigitalTwinGetResponse" and "DigitalTwinUpdateResponse" and we seem to be almost circling back to them. Do you like "DigitalTwinGetResponse" more than "GetDigitalTwinResponse"?

Reply:
I personally do like DigitalTwinGetResponse more than GetDigitalTwinResponse. The latter, to me, looks like an API call.

Reply:
Let's go with that pattern then. DigitalTwinGetResponse and DigitalTwinUpdateResponse